src/Components/Page/Page.js

```diff
@@ -8,11 +8,17 @@ import Spinner from '../../Spinner'
 import { Container } from './Page.styles'
 import { childrenType } from '../../types'
 import Breadcrumb from '../Breadcrumb'
+import { setApplicationTitle } from '../../redux/actions'
 
 export class Page extends Component {
   componentDidMount() {
-    const { title } = this.props
+    const { title, applicationTitle, dispatchApplicationTitle } = this.props
+
     this.renderTitleString(title)
+
+    if (applicationTitle) {
```

Comment:
what's this stuff doing?

Reply:
This sets the title in the menu. So HMPPS | Whereabouts etc. I want to make this more expressive. Maybe I should rename it to menu title?